src/views/shared/SecurityBeacon.js

```diff
@@ -28,14 +43,19 @@ define(['okta', 'util/Animations'], function (Okta, Animations) {
     el.css('background-image', '');
     el.removeClass('new-user undefined-user');
     if (isNewUser) {
+      el.find('.auth-beacon-description').text('no security image');
```

Comment:
I did consider that possibility. We do have a visible callout that says something like "You are connecting to foo.okta.com for the first time". My thought process is that, if we thought that the lack of a security image was important enough to specifically mention to sighted users, it's worth telling a vision-impaired user as well.